.travis_install.sh

```diff
@@ -27,5 +27,5 @@ source activate testenv
 
 pip install -r requires.txt
 conda install -c conda-forge python-igraph louvain --yes
-pip install docutils
+pip install docutils phate
```

Comment:
Phate is an optional dependency and should stay an optional dependency. It's not (yet) one of the canonical visualization methods in the field and not in Scanpy. It's in Scanpy so that people can try it out.

You don't have to provide a test for it... It's tested in it's own package. There's just a small wrapper in Scanpy.